examples/stm32/xively_demo_stm32f4xx_nucleo_eth/Src/xively_client.c

```diff
@@ -38,7 +38,7 @@
  */
 #define XI_ACCOUNT_ID "4d3c7986-8d53-4cf8-903e-7fd30ff63be1"
 #define XI_DEVICE_ID "6ed0ab44-d7e6-47fa-a3de-6d3441e5c577"
-#define XI_PASSWORD "diYY01MoMWFI3fCGTr7gZamtfFIMmedWrm+gkHGtLHc="
+#define XI_DEVICE_PASS "diYY01MoMWFI3fCGTr7gZamtfFIMmedWrm+gkHGtLHc="

```

Comment:
Should we get rid of these credentials? I always seem to forget updating them the first time I run the code, and end up connecting as your device. If multiple people make the same mistake at the same time, they could get confused with all the seemingly random disconnections.